tests/test_models.py

```diff
@@ -3162,12 +3162,14 @@ def test_no_parent(self):
     def test_get_recent_logs(self):
         # Add some logs
         for _ in range(5):
-            self.project.logs.append(NodeLogFactory())
+            self.project.add_log('file_added', params={'node': self.project._id}, auth=self.auth)
+
         # Expected logs appears
         assert_equal(
             self.project.get_recent_logs(3),
-            list(reversed(self.project.logs)[:3])
+            list(reversed(self.project.logs))[:3]
```

Comment:
This should be leveraging ODM's sorting and slicing.
`list(self.project.logs.sort('date')[:3])`